src/shared-components/toggle/style.js

```diff
@@ -17,7 +17,7 @@ export const Container = styled.div`
 `;
 
 export const Label = styled.span`
-  color: ${COLORS.primaryTint1};
+  color: ${({ theme }) => theme.COLORS.primaryTint1};
```

Comment:
We don't need to explicitly pass the themes as props in usage--the `ThemeProvider` handles it for us.